db/password.go

```diff
@@ -49,13 +49,12 @@ func (r *passwordInfoRepo) Create(tx repo.Transaction, pw user.PasswordInfo) (er
 	}
 
 	_, err = r.get(tx, pw.UserID)
-	if err != nil {
-		if err != user.ErrorNotFound {
-			return err
-		}
-	} else {
+	if err == nil {
 		return user.ErrorDuplicateID
 	}
+	if err != nil && err != user.ErrorNotFound {
```

Comment:
here and some other places below: `err != nil` can be deleted, it's always true because of the `if err == nil` above